Client/Frontend/Fakespot/FakespotOptInCardViewModel.swift

```diff
@@ -70,6 +70,15 @@ struct FakespotOptInCardViewModel {
     let secondaryButton: String = .Shopping.OptInCardSecondaryButtonTitle
     let secondaryButtonA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.secondaryButton
 
+    // MARK: Button Actions
+    var dismissViewController: (() -> Void)?
+    var shouldOptIn: (() -> Void)?
```

Comment:
From the name I'd assume this is a boolean. How about we rename it to `onOptIn`?

Reply:
sounds good, updated